app/schema.graphql

```diff
@@ -574,6 +575,34 @@ type ExperimentEdge {
   node: Experiment!
 }
 
+type ExperimentRun implements Node {
+  """The Globally Unique ID of this object"""
+  id: GlobalID!
+  traceId: String
```

Comment:
Eventually will have to resolve this

Reply:
Are you referring to the fact that is it nullable? Right now, it's only being return if it is a valid reference to a trace that exists in the DB.